sagemaker_neo_compilation_jobs/pytorch_vgg19_bn/pytorch-vgg19-bn.ipynb

```diff
@@ -24,7 +24,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!~/anaconda3/envs/pytorch_p36/bin/pip install torch==1.2.0 torchvision==0.4.0"
+    "!~/anaconda3/envs/pytorch_p36/bin/pip install torch==1.2.0 torchvision==0.4.0 "
```

Comment:
[This is for both notebooks]

Pulled in this PR for review and using a an SM NI. After installing torch and torchvision one gets:

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-3-0e453c056efa> in <module>
      1 import torch
----> 2 import torchvision.models as models
      3 import tarfile

~/anaconda3/envs/pytorch_p36/lib/python3.6/site-packages/torchvision/__init__.py in <module>
      1 from torchvision import models
----> 2 from torchvision import datasets
      3 from torchvision import ops
      4 from torchvision import transforms
      5 from torchvision import utils

~/anaconda3/envs/pytorch_p36/lib/python3.6/site-packages/torchvision/datasets/__init__.py in <module>
      7 from .svhn import SVHN
      8 from .phototour import PhotoTour
----> 9 from .fakedata import FakeData
     10 from .semeion import SEMEION
     11 from .omniglot import Omniglot

~/anaconda3/envs/pytorch_p36/lib/python3.6/site-packages/torchvision/datasets/fakedata.py in <module>
      1 import torch
      2 from .vision import VisionDataset
----> 3 from .. import transforms
      4 
      5 

~/anaconda3/envs/pytorch_p36/lib/python3.6/site-packages/torchvision/transforms/__init__.py in <module>
----> 1 from .transforms import *

~/anaconda3/envs/pytorch_p36/lib/python3.6/site-packages/torchvision/transforms/transforms.py in <module>
     15 import warnings
     16 
---> 17 from . import functional as F
     18 
     19 if sys.version_info < (3, 3):

~/anaconda3/envs/pytorch_p36/lib/python3.6/site-packages/torchvision/transforms/functional.py in <module>
      3 import sys
      4 import math
----> 5 from PIL import Image, ImageOps, ImageEnhance, PILLOW_VERSION
      6 try:
      7     import accimage

ImportError: cannot import name 'PILLOW_VERSION'
```

Do none of the Jupyter notebook PyTorch kernels have torch==1.2.0 and torchvision==0.4.0? Or why are we manually installing this?

Reply:
This looks like a pillow version mismatch. What version of Pillow is being installed? `Pillow==7.2.0` seems to be working fine.

Reply:
These installs are unchanged from the previous version of the notebooks. I tested the notebooks on SageMaker notebook instances. Not sure why the imports fail for you.

Reply:
Just checking if the kernel of the notebook is set to "conda_pytorch_p36" @ngluna

Reply:
> This looks like a pillow version mismatch. What version of Pillow is being installed? `Pillow==7.2.0` seems to be working fine. Also, should the PyTorch version be 1.4.0 instead of 1.2.0?

torchvision 0.4.0 requires torch==1.2.0

Reply:
> Just checking if the kernel of the notebook is set to "conda_pytorch_p36" @ngluna

Yup. That is the kernel it is using.